core/trino-main/src/main/java/io/trino/sql/analyzer/Analysis.java

```diff
@@ -570,7 +570,7 @@ public void registerTable(
             String authorization,
```

Comment:
I don't understand the commit message. What `referenced by view` means? Is it mean that is a table is used by view body? Why this information is useful?

Reply:
> Is it mean that is a table is used by view body
Yes .. The table is not a part of the actual query statement but it used by the view. 

> Why this information is useful
It allows us to identify the tables which are specified in a given query.

Reply:
Discussed offline.